mods/stairs/init.lua

```diff
@@ -276,17 +274,144 @@ if replace then
 	})
 end
 
+-- Register stairs.
+-- Node will be called stairs:stair_inner_<subname>
+
+function stairs.register_stair_inner(subname, recipeitem, groups, images, description, sounds)
+	groups.stair = 1
+	minetest.register_node(":stairs:stair_inner_" .. subname, {
+		description = description,
```

Comment:
We can automatically add 'Inner' and 'Outer' to the decriptions to avoid having to add 2 extra descriptions to the stair registrations
`description = description .. " Inner",`
(hope this works)